src/blocks/advance-columns/column/components/inspector.js

```diff
@@ -298,6 +299,7 @@ export default class Inspector extends Component {
       <InspectorControls key="inspector">
         <InspectorTabs>
           <InspectorTab key={"content"}>
+            <Notice isDismissible={false} status="warning"><p>⚠️ {__( 'Heads up! This block will be deprecated soon. We recommend using the Container block instead.', 'responsive-block-editor-addons' )}</p></Notice>
```

Comment:
Use sanitization at both places.
Also add sanitization in section also if not done already